tests/content-type-attachment-test.php

```
@@ -0,0 +1,280 @@
+    return $GLOBALS['slimstat_attachment_test_state'][$key] ?? false;
+}
+
+if (!function_exists('get_post_type')) {
+    function get_post_type()
+    {
+        return slimstat_attachment_test_state('post_type');
+    }
+}
+
+if (!function_exists('get_object_taxonomies')) {
+    function get_object_taxonomies($post)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$post'. (undefined)

(UnusedFormalParameter)